cli/module_graph.rs

```diff
@@ -768,15 +768,16 @@ impl Graph {
       "checkJs": false,
       "emitDecoratorMetadata": false,
       "importsNotUsedAsValues": "remove",
-      "inlineSourceMap": true,
+      "inlineSourceMap": false,
```

Comment:
`inlineSourceMap: true` previously did not have any effect as source maps were not produced before. This will probably be set back to true when we implement source maps for `deno bundle`.

Currently, `Deno.emit` produces a `separate deno://bundle.js.map` file when the bundle option is set.